Release Notes/2019/2019-06-11-cloud-platform-release-notes.md

```diff
@@ -13,7 +13,7 @@
 
 ##### Automated vApp Templates Catalog
 
-All new builds will use an automated Catalog subscription model. The Default Catalog, named CTL-Golden-Images, will come populated with various vApp Templates. For more information, please refer to the knowledge base article [Default Catalog and How to Opt Out](../../Lumen Private Cloud on VMware Cloud Foundation/Catalog/default-catalog.md).
+All new builds will use an automated Catalog subscription model. The Default Catalog, named **(siteID) (data center) cl01 Lumen**, will come populated with various vApp Templates. For more information, please refer to the knowledge base article [Default Catalog and How to Opt Out](../../lumen-edge-private-cloud/catalog/default-catalog.md).
```

Comment:
I see what you were trying to do. The URL routes don't correspond to the folder names in PublicKB. So while the url when viewing https://www.ctl.io/knowledge-base/lumen-edge-private-cloud/catalog/default-catalog is `lumen-edge-private-cloud/catalog/default-catalog` the path you need to use in the articles need to be by the actual folder name. So for this particular one it would be:
```suggestion
All new builds will use an automated Catalog subscription model. The Default Catalog, named **(siteID) (data center) cl01 Lumen**, will come populated with various vApp Templates. For more information, please refer to the knowledge base article [Default Catalog and How to Opt Out](../../Lumen Edge Private Cloud/Catalog/default-catalog.md).
```

Reply:
The change is made as requested.